src/ncp-spinel/SpinelNCPInstance.cpp

```diff
@@ -747,6 +747,67 @@ unpack_dataset(const uint8_t *data_in, spinel_size_t data_len, boost::any &value
 	return ret;
 }
 
+static int
+unpack_thread_network_time(const uint8_t *data_in, spinel_size_t data_len, boost::any& value, bool as_val_map)
+{
+	spinel_ssize_t len;
+	ValueMap entry;
+	std::list<ValueMap> result_as_val_map;
+	std::list<std::string> result_as_string;
+	uint64_t time;
+	int8_t timeSyncStatus;
+
+	int ret = kWPANTUNDStatus_Ok;
+
+	while (data_len > 0)
+	{
+		len = spinel_datatype_unpack(
+			data_in,
+			data_len,
+			(
+				SPINEL_DATATYPE_UINT64_S   // time
+				SPINEL_DATATYPE_INT8_S     // time sync status
+			),
+			&time,
+			&timeSyncStatus
+		);
+
+		if (len <= 0)
+		{
+			ret = kWPANTUNDStatus_Failure;
+			break;
+		}
+
+		if (as_val_map) {
+			entry.clear();
+			entry[kWPANTUNDValueMapKey_TimeSync_Time] = time;
+			entry[kWPANTUNDValueMapKey_TimeSync_Status] = timeSyncStatus;
+			result_as_val_map.push_back(entry);
+		} else {
+			char c_string[500];
+			int index;
+
+			index = snprintf(c_string, sizeof(c_string), "ThreadNetworkTime: %lu, TimeSyncStatus:%d", time, timeSyncStatus);
```

Comment:
use PRIu64 instead of %lu to support 32bit system.

Reply:
Fixed, thanks.